spec/unit/provider/package/zypper_spec.rb

```diff
@@ -154,14 +145,14 @@ def shell_out_expectation!(command, options=nil)
         "zypper --non-interactive --no-gpg-checks install "+
         "--auto-agree-with-licenses emacs=1.0"
       )
-      provider.upgrade_package("emacs", "1.0")
+      provider.upgrade_package(["emacs"], ["1.0"])
     end
     it "should run zypper upgrade without gpg checks" do
```

Comment:
I'm not sure what 
`"should run zypper upgrade without gpg checks"` really offers over `"should run zypper update without gpg checks"` - perhaps the latter should be removed altogether?

(unless one of the two was meant to test the `dist-upgrade, dup Perform a distribution upgrade.` feature of zypper)